src/pages/library/data-catalog/CreateDataset/CreateDataset.test.ts

```diff
@@ -4,8 +4,12 @@ import { h } from 'react-hyperscript-helpers'
 import { CreateDataset } from 'src/pages/library/data-catalog/CreateDataset/CreateDataset'
 
 
-jest.mock('src/libs/nav')
```

Comment:
With all of nav mocked, this test failed because `getCurrentUrl` (called by `getEnabledBrand`) returned undefined.